manager/controllers/app/blueprint_controller.go

```diff
@@ -326,7 +326,9 @@ func (r *BlueprintReconciler) reconcile(ctx context.Context, cfg *action.Configu
 		}
 
 		releaseName := managerUtils.GetReleaseName(managerUtils.GetApplicationNameFromLabels(blueprint.Labels),
-			managerUtils.GetApplicationNamespaceFromLabels(blueprint.Labels), instanceName)
+			managerUtils.GetApplicationNamespaceFromLabels(blueprint.Labels),
```

Comment:
why do we need the namespace? is uuid not enough
 ?

Reply:
It probably is...